### PR TITLE
I43 create popup dialog modal

### LINKED ISSUE #43: Create Popup dialog modal
## Basic Information
This will be a reusble component
- For now put a button to open and close the modal on the home page (the component shouldnt include this button)

Emits and props can help you with this task.

Props that should be passed into the modal component:
- Title
- body text

Emit back the action type. e.g "cancel or delete"

Screen Shot 2023-06-29 at 4 35 05 pm

### PR DESCRIPTION
**Pull request by K-Straiton**
## Change Summary
* Added a popup dialog modal with styling to match the figma.
* Added popup dialogs to confirm adding and editing event in the EventsModal

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related Issue

- Resolve #43

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -18,6 +18,8 @@ coverage
 /cypress/screenshots/
 
 # Editor directories and files
+
+
```
Comment (ChantelleKerr) [inline client/.gitignore:22]: remove these extra lines

File: client/src/components/PopupDialog.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  set(value) {
+    emit('update:modelValue', value)
+  }
+})
+
+const { mobile } = useDisplay()
+</script>
+
+<template>
+  <v-dialog v-model="value" :fullscreen="mobile" max-width="912px">
```
Comment (ChantelleKerr) [inline client/src/components/PopupDialog.vue:21]: We dont need this to be fullscreen for mobile - its just a small popup so it should be fine

File: client/src/components/PopupDialog.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <v-card class="d-flex flex-column">
+      <div style="height: 8px">
+        <v-img src="/images/Footer-min.jpeg" width="100%" cover></v-img>
```
Comment (ChantelleKerr) [inline client/src/components/PopupDialog.vue:24]: remove the ending v-img tag and close it within the first tag

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Looking great! Just some very small changes.

**Comment by K-Straiton** [comment]
Thanks Chantelle. I've made the changes now

**Review by ChantelleKerr — APPROVED** [review]
Great work, looks really good. Approved.

You can merge it into main now :)

**Pull request merged**